### PR TITLE
Add validation section after each step

### PR DESCRIPTION
**Pull request by david-richer-adsk**

### CODE REVIEW

File: docs/en/offerings/spc/setup/traffic_segregation.md [hunk 1]
```diff
@@ -11,13 +11,15 @@ The goal is to set up an AWS PrivateLink to privately access your Shotgun site.
 
 ## Set up PrivateLink to Shotgun
 
-* Ask Shotgun support to provide you with the Shotgun PrivateLink service name for your AWS region.
+  * Ask Shotgun support to provide you with the Shotgun PrivateLink service name for your AWS region.
 
-* Add a new VPC Endpoint in your VPC
+  * Update the private VPC CloudFormation stack you creaed earlier and set ShotgunPrivateServiceName parameter.
```
Comment (comeaug) [inline docs/en/offerings/spc/setup/traffic_segregation.md]: created

File: docs/en/offerings/spc/setup/traffic_segregation.md [hunk 2]
```diff
@@ -30,3 +32,13 @@ Example DNS entry
 ```
 mystudio-staging.shotgunstudio.com   CNAME   vpce-048447456a4f57e14-1j3wh50q.vpce-svc-0b054415458f57634.us-west-2.vpce.amazonaws.com
 ```
+
+## Validation
+
+Verify that your site resolve to IPs in your AWS VPC block.
```
Comment (comeaug) [inline docs/en/offerings/spc/setup/traffic_segregation.md]: resolves

File: docs/en/offerings/spc/setup/traffic_segregation.md [hunk 3]
```diff
@@ -30,3 +32,13 @@ Example DNS entry
...
+
+```
+nslookup mystudio-staging.shotgunstudio.com
+```
+
+Try to access your test site from inside your office ie https://mystudio-staging.shotgunstudio.com
```
Comment (comeaug) [inline docs/en/offerings/spc/setup/traffic_segregation.md]: since you switched staging to test above in the doc, might as well change it here too

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/92a094b3184bd06951768d06ceb789a275844e9f/index.html)

File: docs/en/offerings/spc/setup/traffic_segregation.md (hunk 3 above)
Comment (david-richer-adsk) [reply docs/en/offerings/spc/setup/traffic_segregation.md]: I will keep staging for now.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/92dd2d4bdf232c79cc8606c3f98660e73decfca2/index.html)

**Pull request merged**